README.md

```diff
@@ -20,8 +38,18 @@ To run this plugin on Gitlab CI you may want to run Chrome with `no-sandbox` fla
 module.exports = {
   plugins: [
     ['@snowdog/vuepress-plugin-pdf-export', {
+      // to pass puppeteer launch options
       puppeteerLaunchOptions: {
         args: ['--no-sandbox', '--disable-setuid-sandbox']
+      },
+      // to pass puppeteer page options
```

Comment:
Everything is alright except this little thing. This part is related only to Gitlab CI, it's not a generic example of how to pass some data.

Reply:
okay, let me remove that.

Reply:
done, and resolving this

Reply:
hi! if you could merge it in and make a release, it'd very helpful

Reply:
I'll try to handle that tomorrow

Reply:
Ah, thank you very much ! It'll help me in my own Gitlab pipeline

Reply:
hiya @Igloczek, just doing a friendly bump